README.md

```diff
@@ -4,6 +4,7 @@ Under development.
 
 ## Requirements
 * Python 3.8+
+* Tested on MacOS Catalina and Ubuntu 18.04
```

Comment:
if we need to use Windows later, then we will need to download the VAL binary for Windows and add it to third party. shouldn't be hard, but I want to wait until we can test it until we add it.